hudi-spark-datasource/hudi-spark/src/main/java/org/apache/hudi/HoodieDataSourceHelpers.java

```diff
@@ -76,14 +76,22 @@ public static List<String> listCommitsSince(HoodieStorage storage, String basePa
 
   // this is used in the integration test script: docker/demo/sparksql-incremental.commands
   public static List<String> listCompletionTimeSince(FileSystem fs, String basePath,
-      String instantTimestamp) {
+                                                     String instantTimestamp) {
```

Comment:
Could this method use `listCompletedInstantSince` (to return `Stream<HoodieInstant>`) to avoid code duplication on the same logic?